packages/aragon-cli/test/integration/ipfs/data.test.js

```diff
@@ -48,15 +48,13 @@ test.beforeEach(async t => {
  * Note that the IPFS rpc object is correct, =
  * { protocol: 'https', host: 'ipfs.infura.io', port: 5001 }
  */
-
-/* eslint-disable-next-line ava/no-skip-test */
-test.skip('Get IPFS readme merkle DAG and CIDs', async t => {
+test('Get IPFS readme merkle DAG and CIDs', async t => {
   // arrange
   const { ipfsReader } = t.context
```

Comment:
I don't think that's possible because it's an `async` operation, if we don't put it inside a `before` hook the test will run without waiting for the connection to be established.